### PR TITLE
Update sensitivity-labels-teams-groups-sites.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#978

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/sensitivity-labels-teams-groups-sites.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ To view and edit the labels, use the Active sites page in the new SharePoint adm
 
 ## Change site and group settings for a label
 
-As a best practice, don't change settings after you've applied a label to several teams, groups, or sites. If you must make a change, you need to use an Azure AD PowerShell script to manually apply updates. This method ensures that all existing teams, sites, and groups enforce the new setting.
+As a best practice, don't change settings after you've applied a label to several teams, groups, or sites. If you must make a change, you need to use an Azure AD PowerShell [script](https://docs.microsoft.com/microsoft-365/compliance/sensitivity-labels-teams-groups-sites#enable-this-preview) to manually apply updates. This method ensures that all existing teams, sites, and groups enforce the new setting.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-teams-groups-sites.md]:
```suggestion
As a best practice, don't change settings after you've applied a label to several teams, groups, or sites. If you must make a change, you need to use an [Azure AD PowerShell script](https://docs.microsoft.com/microsoft-365/compliance/sensitivity-labels-teams-groups-sites#enable-this-preview) to manually apply updates. This method ensures that all existing teams, sites, and groups enforce the new setting.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by cabailey** [comment]
#hold-off Looking into this paragraph for consistency and clarity.

**Comment by cabailey** [comment]
@MaratMussabekov The section is now updated with sign off from the PM, so closing this PR.  The script in question is actually to run Set-UnifiedGroup for the updated label, not the commands to enable the preview feature.  It wasn't at all clear, I agree!

**Pull request closed without merging**